README.md

```diff
@@ -64,6 +64,7 @@ to something other than a raw file it should be marked with an asterisk.*
 | [`language_batch`](plugins/language_batch.lua?raw=1)                                                           | Syntax for Windows [Batch Files](https://en.wikipedia.org/wiki/Batch_file)                                                                                                                                                                                                                      |
 | [`language_bib`](plugins/language_bib.lua?raw=1)                                                               | Syntax for [BibTex](https://en.wikipedia.org/wiki/BibTeX) files                                                                                                                                                                                                                                 |
 | [`language_cmake`](plugins/language_cmake.lua?raw=1)                                                           | Syntax for the CMake build system language                                                                                                                                                                                                                                                      |
+| [`language_containerfile`](https://github.com/FilBot3/lite-xl-language-containerfile) | Syntax for [Containerfile](https://github.com/containers/common/blob/main/docs/Containerfile.5.md)/[Dockerfile](https://docs.docker.com/engine/reference/builder/) |
```

Comment:
Would you mind adding an asterisk like this:
```suggestion
| [`language_containerfile`](https://github.com/FilBot3/lite-xl-language-containerfile)\* | Syntax for [Containerfile](https://github.com/containers/common/blob/main/docs/Containerfile.5.md)/[Dockerfile](https://docs.docker.com/engine/reference/builder/) |
```

Reply:
I have added the `*` to the link to denote a repository.